app/routes/events/view/speakers/list.js

```diff
@@ -1,21 +1,13 @@
 import Route from '@ember/routing/route';
 import EmberTableRouteMixin from 'open-event-frontend/mixins/ember-table-route';
+import { capitalize } from 'lodash-es';
 
 export default class extends Route.extend(EmberTableRouteMixin) {
   titleToken() {
-    switch (this.params.speakers_status) {
-      case 'pending':
-        return this.l10n.t('Pending');
-      case 'accepted':
-        return this.l10n.t('Accepted');
-      case 'rejected':
-        return this.l10n.t('Rejected');
-      case 'confirmed':
-        return this.l10n.t('Confirmed');
-      case 'withdrawn':
-        return this.l10n.t('Withdrawn');
-      case 'canceled':
-        return this.l10n.t('Canceled');
+    if (['accepted', 'rejected', 'confirmed', 'withdrawn', 'canceled'].includes(this.params.speakers_status)) {
+      return capitalize(this.params.invoice_status);
```

Comment:
tVar

Reply:
done

Reply:
@iamareebjamal  Should I squash my commits?